README.adoc

```diff
@@ -64,14 +64,69 @@ make setup
 
 NOTE: This is using the `dep` package manager under the hood. You will see the dependencies installed in the `vendor` folder.
 
-=== Start the Server
+=== Configuring the Database
+
+This REST Service is using a https://www.postgresql.org/[PostgreSQL] database. The server will not start without a https://www.postgresql.org/[PostgreSQL] available.
+In order to attend this need has two options; install the database locally or use a docker container with it. Below following the steps.
+
+==== By using the docker image of this project
+
+* Removing any existing docker Postgres containers
+
+[source,shell]
+----
+$ docker rm <container_id>
+----
+
+* Starting the database container
+
+[source,shell]
+----
+$ docker-compose up -d db
+----
+
+NOTE: When the database is started the link:./pkg/db/db.go[db.go] routine will be executed and the database model will be created.
+TIP: You can use the https://www.pgadmin.org/[pgadmin] to work with the database. See link:#installing-and-configuring-client-tool-for-the-database[Installing and configuring client tool for the database] how to setup it in your development environment.
+
+* Checking the database
+
+Following the commands to access the database in the container and check if the tables were created successfully.
```

Comment:
I think we can remove this line. Each step below already details what it's doing.